src/sage/matroids/matroid.pyx

```diff
@@ -7708,15 +7708,13 @@ cdef class Matroid(SageObject):
                     for v in layers[d[u]+1] - visited:
                         # check if edge (u,v) exists in the auxiliary digraph
                         exist = False
-                        if ((u in Y) and
-                            (v in E-Y) and
+                        if ((u in Y) and (v in E-Y) and
                             (self.is_dependent(Y|set([v]))) and
-                            (self.is_independent((Y|set([v])) - set([u])))):
+                                (self.is_independent((Y|{v}) - {u}))):
                             exist = True
-                        if ((u in E-Y) and
-                            (v in Y) and
-                            (not other.is_independent(Y|set([u]))) and
-                            (other.is_independent((Y|set([u])) - set([v])))):
+                        if ((u in E-Y) and (v in Y) and
+                            (not other.is_independent(Y|{u})) and
+                                (other.is_independent((Y|{u}) - {v}))):
                             exist = True
```

Comment:
I don't understand why 7717 is indented while 7716 is not. Why  7713 indented, 7712 not indented?

The original format looks better to me...

Reply:
I have undone the indent change. This is a suggestion of pep8, to avoid confusion with the line *inside* the if statement.

Reply:
I see. I think we should weigh that with the readability of the condition phrase, in this case.

Thanks.